### DISCUSSION TITLE
Git section crashing

### DISCUSSION THREAD

**Discussion by Swat009** [vercel/community#4745] [Bug Report]
### Page to Investigate

https://vercel.com/whatbuild/whatbuild/settings/git

### Steps to Reproduce

I transferred a GitHub repo connected to a project from personal to the organization. Since then, I cannot access the Git section. It crashes, so I can't even disconnect the previous repo link.

<img width="830" alt="Screenshot 2023-11-05 at 6 02 13 PM" src="https://github.com/vercel/community/assets/13027349/a2f0a76f-7951-4251-94ac-09f5ddf7dc28">

**Comment by amyegan** [comment]
Hi @Swat009.  I'm not seeing this error on my test projects. Do you still have this problem?

**Reply by Swat009** [reply]
@amyegan  Yes, I am still experiencing this issue. To recreate it, create a personal repo on GitHub and connect it to your project. Once that's done, transfer this repo to an organization on GitHub. Your Git section will start breaking.

**Reply by Swat009** [reply]
@amyegan just found it is happening only on Safari

**Reply by swarnava** [reply]
Can you confirm if you are using Safari Version 17.1 (19616.2.9.11.7)?

**Comment by K33711647** [comment]
Hey, I am getting this issue in Edge , any update? Thanks.

**Reply by K33711647** [reply]
Ah my error is a bit different: 
![Screenshot 2023-11-13 at 08 53 47](https://github.com/vercel/community/assets/407881/fcb8053c-e9cc-4a41-9c73-10baa40f381d)
